### DISCUSSION TITLE
Watch the KeyValue Store start with a specific revision

### DISCUSSION THREAD

**Discussion by shadow3x3x3** [nats-io/nats-server#4872] [Q&A] [answered]
Hi
I'd like to know if KeyValue Store supports watching from a specific version.

Thanks!

**Comment by ripienaar** [comment]
Are you asking if you can watch starting with a certain revision and only get newer than that revision data?

**Reply by shadow3x3x3** [reply]
Yes. I only care about data that's newer than a specific revision.

**Reply by ripienaar** [reply, marked as answer]
It could be possible but we dont support it in the clients atm

**Reply by shadow3x3x3** [reply]
OK. I look forward to it being implemented in the near future. Thank you.
